src/FFmpegWriter.cpp

```diff
@@ -284,6 +287,39 @@ void FFmpegWriter::SetOption(StreamType stream, string name, string value)
 			// Buffer size
 			convert >> c->rc_buffer_size;
 
+		else if (name == "crf") {
+			// encode quality and special settings like lossless
+			// This might be better in an extra methods as more options
+			// and way to set quality are possible
+			#if  LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(55, 39, 101)
+					switch (c->codec_id) {
```

Comment:
Can we reduce the indention on the switch statement? Seems really far over to the right. Might be another tab vs space indention issue.